include/dbclean.php

```diff
@@ -16,22 +16,48 @@
 load_config('config');
 load_config('system');
 
-update_shadow_copy();
+remove_orphans();
```

Comment:
It's actually simpler than that: endpoints or run scripts should not declare any functions and library files should not execute any code on include.